src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -81,6 +81,7 @@ machine(MachineType:TCC, "TCC Cache")
     I, AccessPermission:Invalid,    desc="Invalid";
     IV, AccessPermission:Busy,      desc="Waiting for Data";
     WI, AccessPermission:Busy,      desc="Waiting on Writethrough Ack";
+    WIB, AccessPermission:Busy,      desc="Waiting on Writethrough Ack; Will be Bypassed";
```

Comment:
minor nit: "desc" should start 1 character further to the left

Reply:
Fixed.